bazel/cargo/wasmtime/wasmtime-bumpalo.patch

```diff
@@ -0,0 +1,9 @@
+diff --git a/src/lib.rs b/src/lib.rs
+index be68365..47c14cd 100644
+--- a/src/lib.rs
++++ b/src/lib.rs
+@@ -1,4 +1,3 @@
+-#![doc = include_str!("../README.md")]
+ #![deny(missing_debug_implementations)]
+ #![deny(missing_docs)]
+ #![no_std]
```

Comment:
Nitpicking, but this file should be named `bumpalo.patch` after the `bumpalo` crate, not `wasmtime-bumaplo.patch` (and `Cargo.toml` needs to be updated).